fe/lib/components/Questionnaire/index.ts

```diff
@@ -13,4 +13,5 @@ export {
   SelectionQuestion,
   FormComponent,
   PrivacyConsent,
+  MutationVariableInput,
```

Comment:
Does this export do anything?

Reply:
Ah, I exported it so ryanair could use the type for the variable. It seems `QuestionnaireCreateInput` is already exported and `MutationVariableInput` could just be derived from "variables" so maybe it's not needed.

Reply:
👌 was just checking if this was important, as I don't think it gets re-exported out of the parent `fe/lib/components`